action.yml

```diff
@@ -39,10 +39,18 @@ inputs:
     description: "Script to run on host machine before rsync"
     required: false
     default: ""
+  SCRIPT_BEFORE_REQUIRED:
+    description: "If true, the action will fail if the before script fails"
+    required: false
+    default: "false"
```

Comment:
Default should be  false -> empty string, `""`.
Currently it's true -> string "false".

Ideally it should be boolean, but that would require more refactoring since only strings were expected until now.
Just changing to empty string should work fine.